file_formats/nwb.md

```diff
@@ -0,0 +1,62 @@
+# Standards on Neurodata Without Borders files
+
+## Version
+
+`0.1.0-draft`
+
+## Introduction
+
+Neurodata Without Borders (NWB) files are a standard file format for neurophysiology data, however there are many topics where NWB is not opinionated. This documents defines AIND's opinions. 
+
+## Basics
+
+We also take this opportunity to remind readers about some general NWB basics:
+
+- Raw data goes in the `acquisition` group
+- Processed data goes in the `processing` group
+- The root level `session_start_time` is a timezone-aware datetime defining the start of acquisition.
+- All timestamps must relative to the same point in time and require no additional alignment to compare. This is defined in the root level `timestamps_reference_time`, which defaults to `session_start_time`.
+- It is not required that timestamps for different data streams be identical (i.e. resample data such that all timeseries to have exactly the same timestamps).
+
+## HARP
+
+- At AIND, if there is a HARP board available, timestamps must utilize HARP TTLs to transform timestamps to a common timebase. It is insufficient to store a lookup table.
```

Comment:
If a primary data asset is being packaged (e.g. FrameTime_gpu_clock -> StimulusParameters), should we package the "raw" form where no alignment has been done yet, OR convert to something along the lines of "FrameTime_harp_clock -> StimulusParameters"? There are trade-offs, the most obvious one is that data consumers will need to "trust" the provided alignment and wont be able to QC on their end.

Reply:
I wouldn't opposed storing both sets of timestamps if the alignment process is unreliable, unless there are significant storage implications.

Reply:
@bruno-f-cruz take a look - I added some language